tools/clang/test/hlsl_cmds_test/test_rs.test

```diff
@@ -0,0 +1,18 @@
+// Root Signature target
+// RUN:%echo #define main "CBV(b0)"> %t.test-global-rs.hlsl
```

Comment:
Why not just have this in a file that serves as the test file?

Reply:
Was trying to keep what the original test doing.
Will just create the file.